packages/framework/src/Form/Admin/Transport/TransportFormType.php

```diff
@@ -36,7 +37,11 @@ public function __construct(VatFacade $vatFacade)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder
+        $builderBasicInformationGroup = $builder->create('information', GroupType::class, [
```

Comment:
Ok, for me this is really wierd, because thats how you map properites of entity into form so is does not make much sense for me to use camelCase for just group names, but ok.

Reply:
Every name of form element is camelCase and even every form group created before this PR had the name in camelCase. That is the reason why I mentioned it here.